test/testutils.jl

```diff
@@ -46,5 +46,5 @@ function test_clf(m, X, y)
     else
         ŷ = MB.predict_mode(m, f, X)
     end
-    return MB.accuracy(ŷ, y), f
+    return StatisticalMeasures.accuracy(ŷ, y), f
```

Comment:
Maybe this is the only time we use a measure? If so, we could do

```suggestion
    return sum(ŷ .== y)/length(y), f
```
and drop the dependency on StatisticalMeasures ??

Reply:
Done!